zuul-sample/src/main/java/com/netflix/zuul/sample/ZuulSampleModule.java

```diff
@@ -45,6 +47,13 @@
 public class ZuulSampleModule extends AbstractModule {
     @Override
     protected void configure() {
+        try {
+          ConfigurationManager.loadCascadedPropertiesFromResources("application");
+        } catch (Exception ex) {
+          throw new RuntimeException("Error loading configuration: " + ex.getMessage());
```

Comment:
Not to be nitpicky, but what I meant was that it needs to chain the exceptions together, to make sure the original stack trace isn't lost.